numpy/core/tests/test_multiarray.py

```diff
@@ -1760,32 +1796,34 @@ def test_sort(self):
                 msg = 'byte-swapped complex sort, dtype={0}'.format(dt)
                 assert_equal(c, arr, msg)
 
-        # test string sorts.
-        s = 'aaaaaaaa'
-        a = np.array([s + chr(i) for i in range(101)])
+    @pytest.mark.parametrize('message, s, dtype',  [('string', 'aaaaaaaa', 'str'),('unicode', 'aaaaaaaa', np.unicode_)])
+    def test_sort_string(self, message, s, dtype):
+        a = np.array([s + chr(i) for i in range(101)], dtype = dtype)
```

Comment:
Something is wrong with these tests, `string` and `unicode` are the same type in python 3.

Reply:
Not that that needs to be fixed in this patch, but if you're interested it would be good to tackle it.